chord_metadata_service/restapi/authz_middleware.py

```diff
@@ -2,10 +2,21 @@
 from django.http import HttpResponseServerError
 import requests
 import json
+import os
+import re
 
 REQUEST_PATHS_TO_AUTHZ_PATHS = {"/api/phenopackets": "api/phenopackets",
                                 "/api/individuals": "api/ga4gh/individuals"}
 
+AUTHZ_PATHS = ["^/api/phenopackets/?.*", "^/api/datasets/?.*", "^/api/diagnoses/?.*", "^/api/diseases/?.*",
+               "^/api/genes/?.*", "^/api/genomicinterpretations/?.*", "^/api/htsfiles/?.*", "^/api/individuals/?.*",
+               "^/api/interpretations/?.*", "^/api/metadata/?.*", "^/api/phenopackets/?.*",
+               "^/api/phenotypicfeatures/?.*", "^/api/procedures/?.*", "^/api/variants/?.*"]
+rootCA = os.getenv("ROOT_CA", None)
+CANDIG_OPA_VERSION = os.getenv("CANDIG_OPA_VERSION", "dycons")
+permissions_secret = os.getenv("PERMISSIONS_SECRET",
```

Comment:
Since these 3 variables are environment variables, I think we should be consistent with the other environment variables and make these variables all caps (`rootCA` -> `ROOT_CA`, `permissions_secret` -> `PERMISSIONS_SECRET`). 

Also, the way the other environment variables are loaded into the program is in the file `chord_metadata_service/metadata/settings.py`. Basically, `settings.py` loads the `.env` file and loads the variables like this:
```
CANDIG_OPA_URL = os.environ.get("CANDIG_OPA_URL")
```
Then, in other files you can access the variable like this: `settings.CANDIG_OPA_URL`.
To be consistent with the other environment variables, I think these 3 variables should be refactored to this way.

Also, these 3 environment variables should be added to the file `.env-sample`.